Makefile

```diff
@@ -4,13 +4,18 @@
 #
 # Authors:
 #   Jan Kiszka <jan.kiszka@siemens.com>
+#   Li Huaqian <huaqian.li@siemens.com>
 
 CROSS_COMPILE = arm-linux-gnueabihf-
 CC = gcc
 
 RTI_MODULE = 1
+PON_REASON_BASE_ADDR = 0xA2200000
 
 CFLAGS = -mcpu=cortex-r5 -DRTI_MODULE=$(RTI_MODULE)
+ifdef PON_REASON_BASE_ADDR
```

Comment:
Given that we now have a default value for PON_REASON_BASE_ADDR again, this test is always true, isn't it?

Reply:
Yes. What is your suggestion? Another way to set the default value, or roll back the codes to no macros?

Reply:
See comment below: We need a value that means "disabled", but even then `PON_READ_BASE_ADDR` will always be defined at this level.